packages/wonder-blocks-core/src/hooks/use-latest-ref.js.flow

```diff
@@ -0,0 +1,14 @@
+/**
+ * Flowtype definitions for useLatestRef
+ * @flow
+ */
+
+/**
+ * The useLatestRef hook returns a ref that always contains the `value` passed
+ * to the hook during the most recent render.
+ *
+ * It can be used to wrap a possibly-changing prop in a stable value that can
+ * be passed to useEffect without causing unnecessary re-renders. See the
+ * Storybook docs for a usage example.
+ */
+declare export function useLatestRef<T>(value: T): {|current: T|};
```

Comment:
**nit:** I believe we do have a type for this in Flow - but this should be fine as-is.